rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/executor/BakePollerSpec.groovy

```diff
@@ -129,4 +136,72 @@ class BakePollerSpec extends Specification {
       1 * bakeStoreMock.retrieveBakeStatusById(JOB_ID) >> new BakeStatus()
   }
 
+  void 'Decorate the bakeDetails with an artifact if bake is successfull'() {
```

Comment:
ah, I always turn of spellcheck when coding. Because it trips over my variables and methods. I guess it's time to put it back on.